fastlane/Deliverfile

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal:true
+
```

Comment:
There's a bunch of RuboCop styling fixes here. 

I tried to use `deliver` to go around the issue with App Store Connect rejecting locales because "the app name is already used in this account" but it didn't work out. That's not surprising, but it was worth a shot.

I thought it would be useful to track these changes regardless.